src/sentry/static/sentry/app/components/events/interfaces/frame.jsx

```diff
@@ -665,4 +669,11 @@ const HintStatus = styled(InlineSvg)`
   transform: translateY(-1px);
 `;
 
+const Symbol = styled('span')`
+  text-align: center;
```

Comment:
The current percy diffs show all the text center aligned. Should they be left aligned?

Reply:
IMO center-aligned text is better on mobile